client/scripts/DrawTogether.js

```diff
@@ -1211,6 +1216,32 @@ DrawTogether.prototype.createDrawZone = function createDrawZone () {
 	// locationImage.src = "images/icons/locations.png";
 	// locationImage.alt = "Locations";
 	// locationImage.title = "Locations";
+	
+	//Favorites button 
+	var favoritesButton = this.paint.coordDiv.appendChild(document.createElement("div"));
+	favoritesButton.className = "control-button favorites-button";
+	
+	var favoritesButtonImage = favoritesButton.appendChild(document.createElement("img"));
+	favoritesButtonImage.src = "images/icons/locations.png";
+	favoritesButtonImage.alt = "Open Favorites Menu";
+	favoritesButtonImage.title = "Open Favorites Menu";
+	favoritesButton.addEventListener("click", function () {
+		if($(".favorites-window").is(":visible"))
+    {
```

Comment:
my local copy doesnt have that issue, im not sure why it's doing that on that line